api/src/main/java/edu/cornell/mannlib/vitro/webapp/config/ConfigurationPropertiesSmokeTests.java

```diff
@@ -72,9 +74,35 @@ private void checkDefaultNamespace(ServletContext ctx,
 		}
 	}
 
+	/**
+	 * Warn if runtime.properties exists in multiple locations
+	 * or is located vivo.home instead of vivo.home/config
+	 */
+	private void checkMultipleRPFs(ServletContext ctx,
+			ConfigurationProperties props, StartupStatus ss) {
+		String rpfStatus = props.getProperty(MULTIPLE_RPFS);
+
+		if (rpfStatus == "both") {
```

Comment:
.equals() would be more robust than == 
This will work because you used an interned string in ConfigurationPropertiesSetup.